db/cyndi_setup_test.sql

```diff
@@ -7,6 +7,8 @@
 CREATE SCHEMA IF NOT EXISTS inventory;
 
 -- This table should never be queried direcly
+DROP TABLE IF EXISTS inventory.hosts_v1_1 CASCADE;
```

Comment:
@akofink I'm a bit concerned about this, but I guess this is safe in test :thinking: :shrug:

Reply:
We should probably just use `CREATE OR REPLACE TABLE ...`

Reply:
Yup